test/JDBC/expected/restricting_permission_on_role.out

```diff
@@ -0,0 +1,167 @@
+-- tsql
```

Comment:
Although I think we should add these tests into existing `ownership_restrictions_from_pg` flle but if you really want to have a new file then I would recommend to name this as `permission_restrictions_from_pg` to keep the naming style similar.